--- build-b-files
+++ build-c-files
@@ -3,14 +3,14 @@
 checkpoint-fixtures/arm64/case-00-split-002.hvgcp	3b0854041840c7bc38ab29ba54d6958a066a7f2b52086a6a0895d2767c572cd5	412638	file
 checkpoint-fixtures/arm64/case-00-split-003.hvgcp	4f964de88ede61fa40495fb4ee57ebddb63f07384101d50c11fcabee4d906930	412639	file
 checkpoint-fixtures/arm64/case-00-split-004.hvgcp	f87c6637b69ab2afedb5d949439c9f6c5758dfc4dc4c5501b605ca84711c6a5f	412640	file
-checkpoint-fixtures/arm64/case-00-split-005.hvgcp	3c8cd8bfd9cdb191cf28afa8da54bec294d9bfbdad154ac5939048c9981acf45	412636	file
-checkpoint-fixtures/arm64/case-00-split-006.hvgcp	7f0f80e8e60cd456187e1008fe7a7b46ce4ff937946b15936fbe0efb370d42ca	412636	file
-checkpoint-fixtures/arm64/case-00-split-007.hvgcp	a1c69875e79e997615fd96804c10dddb20736983fedd5460f3f18a4d4fa5e9ea	412636	file
-checkpoint-fixtures/arm64/case-00-split-008.hvgcp	e805f6d6de714eb92c5fb88a8147f04b7cd168b8d90825f8029710b6f945f906	412636	file
-checkpoint-fixtures/arm64/case-00-split-009.hvgcp	3e5eeb150e5f0b8c85c49e3e845d0bd7c275b2565f5245dc9a1e84e5764f4ea0	412637	file
-checkpoint-fixtures/arm64/case-00-split-010.hvgcp	940878fb9803b2decc300f8ba6e258c5739d48779c8391849df95ce803ef1281	412638	file
-checkpoint-fixtures/arm64/case-00-split-011.hvgcp	9a668da1f2ecf3df46516eb86bf6f1bd3bac6eb5e7e1b802c359282213da8987	412639	file
-checkpoint-fixtures/arm64/case-00-split-012.hvgcp	823829d8f8d019c03fa854891e99b0085c1f4aa80e834c3af99dd67737a4a482	412636	file
+checkpoint-fixtures/arm64/case-00-split-005.hvgcp	b078a0d80b97a23a4265d47e1e872e751f26e7f5009f3ecae9188b5fc07a7e55	412636	file
+checkpoint-fixtures/arm64/case-00-split-006.hvgcp	c2c21bd73afb65fac7e2549d087e287ead76b79fb2799dff98d9a20b2e781a91	412636	file
+checkpoint-fixtures/arm64/case-00-split-007.hvgcp	492990f1488e940a22f33f6a11ef267942652b00e2ad1035a8e62422b104715f	412636	file
+checkpoint-fixtures/arm64/case-00-split-008.hvgcp	bf44577850f4eceddbebb027f6370056575dae2dbd0f08f5d3032e1c675956f9	412636	file
+checkpoint-fixtures/arm64/case-00-split-009.hvgcp	033d0cc4749e575e9880bfce9b6756bd9ef460e97d94aa036bdee3f105eae00a	412637	file
+checkpoint-fixtures/arm64/case-00-split-010.hvgcp	e7fb527bffc98e896c0d56c466c815a16b081680fbf7f530ff6a69a628b64c96	412638	file
+checkpoint-fixtures/arm64/case-00-split-011.hvgcp	3e5e8463ade11457bf3c80678709966e3f40c17b746a64cceed99bd455aa2f44	412639	file
+checkpoint-fixtures/arm64/case-00-split-012.hvgcp	a8f152b44459f251cd18f346fc1cae93e976842be1ac62986f7e0a8f3f46cb0d	412636	file
 checkpoint-fixtures/arm64/case-01-split-000.hvgcp	17397ebde0f0a4197a3130d9efeb4c0184bdee4a74be4244331a4825a5f66b74	412636	file
 checkpoint-fixtures/arm64/case-01-split-001.hvgcp	ef61f2544f5e81213603e0e65eac08d329b1a30202cfc6a047f024fc9b4528b2	412637	file
 checkpoint-fixtures/arm64/case-01-split-002.hvgcp	77a24381289d454c58260e89eca310bb841dcc396ccfe8c165ba249a9a605e0c	412638	file
