--- build-b-files
+++ build-c-files
@@ -3,14 +3,14 @@
 checkpoint-fixtures/arm64/case-00-split-002.hvgcp	680546a7c992465ab55d80fd38c951997341aff8e48f43f557441da35a33e624	412638	file
 checkpoint-fixtures/arm64/case-00-split-003.hvgcp	b8d5eb3ac2fb7dd3800ccac467a34b8e59b92dc001efd1ab5ad0f8f750895298	412639	file
 checkpoint-fixtures/arm64/case-00-split-004.hvgcp	c308ac5bbeb309d49c1ef678f7f2957482a7f2778f92d84b064855b46c2ee77a	412640	file
-checkpoint-fixtures/arm64/case-00-split-005.hvgcp	8c2993b209152c2571f854ddbdee93aa305deef0c21d9c8852c7a1848fc774fa	412636	file
-checkpoint-fixtures/arm64/case-00-split-006.hvgcp	333de49036e4aa7942a8756b0deb4ccc9c7e0494bdc04fc425e76c5e0e04db62	412636	file
-checkpoint-fixtures/arm64/case-00-split-007.hvgcp	d843b3c15d28cbaf7aa832bf2042a59d09cb24faec64733a962f1e017c35de2b	412636	file
-checkpoint-fixtures/arm64/case-00-split-008.hvgcp	a11098afa44b2872f792faaecfb3d9ce673bf1730673b198d7214bb136574252	412636	file
-checkpoint-fixtures/arm64/case-00-split-009.hvgcp	b386d23abf786a0e5b8ef5e0a7c9a35f79c02613744ee60ac7cf979c351af350	412637	file
-checkpoint-fixtures/arm64/case-00-split-010.hvgcp	753d61e6c3b118b4056a291e6016915fe585273ea3a359d16c9ef48186b9ec9c	412638	file
-checkpoint-fixtures/arm64/case-00-split-011.hvgcp	1ba922a419737679ce19b3bbcc2c472f7f0d3826543edf521361c8ac96516eac	412639	file
-checkpoint-fixtures/arm64/case-00-split-012.hvgcp	02c1746a503cabc209b799174803af9f765a446fe08bba4cfcff9b115817b68c	412636	file
+checkpoint-fixtures/arm64/case-00-split-005.hvgcp	7fc73a2dd4345a0d12162a988e2e5e9ca3a0ca05090f830d684d826082206d6f	412636	file
+checkpoint-fixtures/arm64/case-00-split-006.hvgcp	0115c8b488a8cd3c59cca9034221825bec5a2ee9c74a4c765363531c02a3e623	412636	file
+checkpoint-fixtures/arm64/case-00-split-007.hvgcp	62ae7677ad9e45043842c9810d0989af598b0196e1547d706f23c58565e30935	412636	file
+checkpoint-fixtures/arm64/case-00-split-008.hvgcp	ed6a00f4b90e6b45c77878438bc5e6d5395b65b140bc1bebcd037c55c295cc12	412636	file
+checkpoint-fixtures/arm64/case-00-split-009.hvgcp	12e9f46b41824ee420045dff69c502b201b718191b45110df3ee871ba333ec09	412637	file
+checkpoint-fixtures/arm64/case-00-split-010.hvgcp	386b2884f205009568caecb15ae00fac92e42c0a11833f4a452bff8e9fd67dbb	412638	file
+checkpoint-fixtures/arm64/case-00-split-011.hvgcp	967dcdead489c29f5d5f403b05292f0cf0b99a52d83ce564b3dee9410dbf4557	412639	file
+checkpoint-fixtures/arm64/case-00-split-012.hvgcp	ca9d8cef071c23fd976a4afea855e070d450ec1b565a1c769f969bb06c138736	412636	file
 checkpoint-fixtures/arm64/case-01-split-000.hvgcp	ff2b0d7b945bbfce87d59226a14da8700375bc64f24811933d8d1ecb120d62c1	412636	file
 checkpoint-fixtures/arm64/case-01-split-001.hvgcp	5d493e03f0e105b28520c161a59d22892675b10b7a3f163fc813d836a74276ac	412637	file
 checkpoint-fixtures/arm64/case-01-split-002.hvgcp	12b64c4fb2031c9d3341c31bf7a513edcc57a54ad4dc0e2a6fc4d27f3af5e8a5	412638	file
