--- build-b-files
+++ build-c-files
@@ -3,14 +3,14 @@
 checkpoint-fixtures/arm64/case-00-split-002.hvgcp	680546a7c992465ab55d80fd38c951997341aff8e48f43f557441da35a33e624	412638	file
 checkpoint-fixtures/arm64/case-00-split-003.hvgcp	b8d5eb3ac2fb7dd3800ccac467a34b8e59b92dc001efd1ab5ad0f8f750895298	412639	file
 checkpoint-fixtures/arm64/case-00-split-004.hvgcp	c308ac5bbeb309d49c1ef678f7f2957482a7f2778f92d84b064855b46c2ee77a	412640	file
-checkpoint-fixtures/arm64/case-00-split-005.hvgcp	8c34ba7b07dedd685d80fc7f3045e1b710bd30912bbd0f4cd0824261e48d8a7e	412636	file
-checkpoint-fixtures/arm64/case-00-split-006.hvgcp	4bb3d2caac81a403a149b749a6b21e048e1f17aa96e782f7bed1779fcb627bca	412636	file
-checkpoint-fixtures/arm64/case-00-split-007.hvgcp	661de0afe25f7f3e45f9dd99fc3ebad4a26e3f144b24b28115b91303ca09ff66	412636	file
-checkpoint-fixtures/arm64/case-00-split-008.hvgcp	87db05905d7657385a40e485fa9d34efc10098816cbab9065f190ddcfe0a3ad2	412636	file
-checkpoint-fixtures/arm64/case-00-split-009.hvgcp	00bc0e98b986b7baf632ca5ba709fc36a9909d2f5fe8bf0b4889a806ad4e422c	412637	file
-checkpoint-fixtures/arm64/case-00-split-010.hvgcp	9e62c4f4f1147fdafb06a8482b216f8929d245f1af20d92c1d245881bde91866	412638	file
-checkpoint-fixtures/arm64/case-00-split-011.hvgcp	92547b9f1db6f6c722e479c9143e94674a438b564476fcbc6295525301300746	412639	file
-checkpoint-fixtures/arm64/case-00-split-012.hvgcp	6bde187daaa95d01459fa6596565e354685f4e6f62c27159e5443513022bb3d5	412636	file
+checkpoint-fixtures/arm64/case-00-split-005.hvgcp	483df7f43ac1b2bd4f3482b1c872af1571a759d6780cf45e4ab5cd1820b78d4f	412636	file
+checkpoint-fixtures/arm64/case-00-split-006.hvgcp	4990a595e14637784ebf7bf792649925ebe91789292ccd26bff1186de51dbfde	412636	file
+checkpoint-fixtures/arm64/case-00-split-007.hvgcp	4f34039d23499df2894fa3d9038d84bc850b4a96529c678c702aa6e8d0e35e4f	412636	file
+checkpoint-fixtures/arm64/case-00-split-008.hvgcp	546744f76347cd92adfb161af330fc5b194ad31989be932e81488a80214441d0	412636	file
+checkpoint-fixtures/arm64/case-00-split-009.hvgcp	60a9678b60e1c8b33a150491efc8754a8a46b51a81bdfc5d28c815e3cf1488a9	412637	file
+checkpoint-fixtures/arm64/case-00-split-010.hvgcp	8eba43a504113bc3ab9a9b8c79a9b5b22badb6984af5ef84a9ed3e88623c839e	412638	file
+checkpoint-fixtures/arm64/case-00-split-011.hvgcp	af5f1d5ac95b1e896b69e35214ca188173bc648d1ad8f91e91b110a645b89df3	412639	file
+checkpoint-fixtures/arm64/case-00-split-012.hvgcp	b317b1ad577e0de3e2b9fb096b213375a79a2fe0da6f050a6f648c5227e8b06c	412636	file
 checkpoint-fixtures/arm64/case-01-split-000.hvgcp	ff2b0d7b945bbfce87d59226a14da8700375bc64f24811933d8d1ecb120d62c1	412636	file
 checkpoint-fixtures/arm64/case-01-split-001.hvgcp	5d493e03f0e105b28520c161a59d22892675b10b7a3f163fc813d836a74276ac	412637	file
 checkpoint-fixtures/arm64/case-01-split-002.hvgcp	12b64c4fb2031c9d3341c31bf7a513edcc57a54ad4dc0e2a6fc4d27f3af5e8a5	412638	file
